mods/lord/Blocks/lord_skeleton/init.lua

```diff
@@ -0,0 +1,127 @@
+minetest.mod(function(mod)
+	local S = minetest.get_mod_translator()
+
+	local title_sbmg = S("Remains of ancient miner. Scull and bones")
+	local title_spmg = S("Remains of ancient miner. Scull and pick")
+	local title_sb = S("Skeleton. Scull and bones")
+	local title_sp = S("Skeleton. Scull and pick")
+
+	local common_definition = {
+		node_box = { type = "fixed", fixed =
+			{-0.5, -0.5, -0.5, 0.5, -0.1, 0.5},
+		},
+		selection_box = { type = "fixed", fixed =
+			{-0.5, -0.5, -0.5, 0.5, -0.1, 0.5},
+		},
+		visual_scale = 0.95,
+		groups = {oddly_breakable_by_hand = 3, falling_node = 1},
+		drawtype = "mesh",
+		paramtype2 = "facedir",
+	}
+
+	local common_items = {
+		--{items = {"lord_skeleton:scull_bones"}},
+		{items = {"lottother:narya"},			rarity = 100/2},
+		{items = {"lottother:vilya"},			rarity = 100/0.5},
```

Comment:
Code-Style (по всему массиву)
```suggestion
		{ items = { "lottother:narya"},             rarity = 100/2 },
		{ items = { "lottother:vilya"},               rarity = 100/0.5 },
```